src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -8,6 +8,9 @@ import noop from 'lodash/noop';
 import classNames from 'classnames';
 import { FormattedMessage, injectIntl } from 'react-intl';
 import commonMessages from '../../../../common/messages';
+import messages from '../../../common/messages';
```

Comment:
Should we rename these to something like `commonMessages`? Or should we import these into the `./messages` bundle and re-export them to cut down on the number of imports in the component?